File: 1552-build-an-array-with-stack-operations/build-an-array-with-stack-operations.cs
public class Solution {
    public IList<string> BuildArray(int[] target, int n) {
         List<string>values = new List<string>();
         for(int i=1;i<=target[target.Length-1];i++){
            values.Add("Push");
            if(!target.Contains(i)){
                values.Add("Pop");
            }
         }
            return values;
    }
}

File: 1552-build-an-array-with-stack-operations/build-an-array-with-stack-operations.js
/**
 * @param {number[]} target
 * @param {number} n
 * @return {string[]}
 */
var buildArray = function(target, n) {
   let array = [];
   for(let i=1;i<=n;i++){
     array.push("Push")
    if(!target.includes(i)){
        array.push("Pop")
    }
        if(i==target[target.length-1]) break;
   }
    return array;
};